### DISCUSSION TITLE
How can i search collection's products with filter newest.

### DISCUSSION THREAD

**Discussion by TheLucDev** [vendurehq/vendure#1956] [Q&A]
How can i search collection's products with filter newest. I have a query search with this , and i wanna sort by createdAt but its only have price and name. I've tried with filter but its dont have in this search query. Can u help me , thank u so much !
![Screen Shot 2022-12-30 at 11 09 28](https://user-images.githubusercontent.com/83877114/210033687-f69dccaf-9e9a-47b0-8a89-185c0d4dec29.png)
![Screen Shot 2022-12-30 at 11 09 49](https://user-images.githubusercontent.com/83877114/210033697-16143f4c-3181-4e45-ba1c-61823d474e7f.png)
![Screen Shot 2022-12-30 at 11 10 02](https://user-images.githubusercontent.com/83877114/210033703-867b100d-f670-4561-9594-55f9dc93718a.png)

**Comment by michaelbromley** [comment]
Hi!
This is currently not supported out of the box. But if you use the [Elasticsearch plugin](https://www.vendure.io/docs/typescript-api/elasticsearch-plugin/), you can implement this using the [extendSearchInputType](https://www.vendure.io/docs/typescript-api/elasticsearch-plugin/elasticsearch-options/#extendsearchinputtype) feature.

**Reply by TheLucDev** [reply]
Thank you so much <3
